src/server/mod.rs

```diff
@@ -63,7 +63,7 @@ macro_rules! build_app {
             .wrap(ErrorHandlers::new().handler(StatusCode::NOT_FOUND, ApiError::render_404))
             // These are our wrappers
             .wrap(middleware::precondition::PreConditionCheck::new())
-            .wrap(middleware::db::DbTransaction::new())
+            // .wrap(middleware::db::DbTransaction::new())
```

Comment:
Should we kill this or is my overwhelming paranoia starting to rub off on y'all?

Reply:
Oh ya, I meant to delete that after commenting it out. There's also #693

Reply:
Leave it for the followup bug. This pr is big enough as is.